boards/kernel_layout.ld

```diff
@@ -185,9 +185,10 @@ SECTIONS
     PROVIDE_HIDDEN (__exidx_end = .);
 
     /* Region for on-chip kernel non-volatile storage.
-     * Align on 512 bytes since that is the page size.
-     * Volumes within this region are allocated with the
-     * storage_volume! macro in utils.rs. */
+     *
+     * Align on 512 bytes since that is the page size. Volumes within this
+     * region are allocated with the storage_volume! macro in utils.rs.
+     */
```

Comment:
not really related to this PR, but it is not generally the case that the page size is 512 bytes. That is true for the sam4l, but  the stm32f303 has 2kB pages. Perhaps we should make this configurable somehow?

Reply:
Yeah, we should.

Reply:
Opened #2006